src/helpers/env.ts

```diff
@@ -74,6 +74,31 @@ export const isProd = (
   });
 };
 
+/**
+ * getBranchName returns the suspected branch name for this environment by
+ * searching through a set of common environment variables.
+ *
+ * This could be used to determine if we're on a branch deploy or not, though it
+ * should be noted that we don't know if this is the default branch or not.
+ */
+export const getBranchName = (
```

Comment:
Hey @jpwilliams - I see this is declared, but it's not used in the code - should we be using this in the `Inngest.ts`?

Reply:
Also, should this function be called `getEnvironmentName`? I know the provider-specific env vars are for branches, but that's kind of an implementation detail to our users. Also, the first variable we check is `INNGEST_ENV` so it's a little unintuitive to have it considered in a "branch" function

Reply:
Hi chaps - yeah, this is from the end of last week where the implementation was a little fuzzy and I had removed this functionality. See 5049a6412a8d662a5f8f65ff992e59803fa0fafd.

Re-adding this now with the new spec.